src/common/dom/apply_themes_on_element.ts

```diff
@@ -110,7 +110,7 @@ const processTheme = (
     keys[prefixedKey] = "";
 
     // Try to create a rgb value for this key if it is not a var
-    if (!value.startsWith("#")) {
+    if (!value || !value.startsWith("#")) {
```

Comment:
In what situation is this undefined?

You should use `value?.startsWith(...`

Reply:
Really, this PR is pointless. I was having trouble getting changing the font weight to work, but then I figured out that if I wrapped it in quotes it worked fine. I'm going to go ahead and add that in and edit.

Reply:
Anyway, added that and that.